client/js/tracking.js

```diff
@@ -11,9 +11,19 @@ export default {
       const componentName = component.getAttribute('data-component-name');
       const componentId = component.getAttribute('data-component-id');
       const clickedProperties = JSON.parse(clicked.getAttribute('data-track-click'));
-      const properties = Object.assign({}, clickedProperties, {componentId, action: 'click'});
-
-      trackEvent({name: componentName, properties});
+      const properties = Object.assign(
+        {},
+        clickedProperties,
+        {componentId, action: 'click'}
+      );
+      const ariaExpanded = clicked.getAttribute('aria-expanded');
+      const propertiesWithAria = ariaExpanded ? Object.assign(
+        {},
+        properties,
+        {ariaExpanded}
```

Comment:
This is not very generic, could we not pass this in when calling the track event?